### PR TITLE
optimize: repackage name to org.apache.seata

### PR DESCRIPTION
**Pull request by slievrly**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

repackage name to org.apache.seata

### CODE REVIEW

File: all/pom.xml [hunk 1]
```diff
@@ -644,7 +644,8 @@
                             <createDependencyReducedPom>true</createDependencyReducedPom>
                             <artifactSet>
                                 <includes>
-                                    <include>io.seata:*</include>
+                                    <include>org.apache.seata:*</include>
+                                    <include>io.seata:seata-all</include>
```
Comment (funky-eyes) [inline all/pom.xml:648]: 现在无法确定io.seata:seata-all这个依赖会存在，需要现在就增加相关配置吗？
Can't be sure that the dependency io.seata:seata-all will exist now, do I need to add the relevant configuration now?

**Review by xingfudeshi — APPROVED** [review]
LGTM

**Comment by funky-eyes** [comment]
image
测试通过

File: all/pom.xml (hunk 1 above)
Comment (slievrly) [reply all/pom.xml:648]: In the subsequent PRs, you will see the corresponding compatibility packaging solutions.

**Pull request merged**